CONTRIBUTING.md

```diff
@@ -1,35 +1,211 @@
-# Filing an issue
+# How to contribute to qBittorrent
 
-### Must read
-* If you aren't sure, you can ask on the [**forum**](http://forum.qbittorrent.org) or read our [**wiki**](http://wiki.qbittorrent.org) first.
-* Do a quick **search**. Others might already reported the issue.
-* Write in **English**!
-* Provide **version** information: (You can find version numbers at menu `Help -> About -> Libraries`)
-  ```
-  qBittorrent:
-  Qt:
-  libtorrent:
-  boost:
-  OS version:
-  ```
-* Provide **steps** to reproduce the problem, it will be easier to pinpoint the fault.
-* **Screenshots**! A screenshot is worth a thousand words. just upload it. [(How?)](https://help.github.com/articles/file-attachments-on-issues-and-pull-requests)
+There are three main ways to contribute to the project.
+Read the respective section to find out more.
 
-### Good to know
-* **Be patient**. The dev team is small and resource limited. Devs finding their free time, analyzing the problem and fixing the issue, it all takes time. :clock3:
-* If you can code, why not become a **contributor** by fixing the issue and open a pull request? :wink:
-* Harsh words or threats won't help your situation. What's worse, your complain will (very likely) to be **ignored**. :fearful:
+#### Table Of Contents
+
+[Submitting an issue/bug report](#submitting-an-issuebug-report)
+  * [Bug reporting etiquette](#bug-reporting-etiquette)
+  * [What is and actual bug report?](#what-is-and-actual-bug-report)
+  * [Before submitting a bug report](#before-submitting-a-bug-report)
+  * [Steps to ensure a good bug report](#steps-to-ensure-a-good-bug-report)
+
+[Suggesting Enhancements/feature requests](#suggesting-enhancementsfeature-requests)
+  * [Before submitting an enhancement/feature request](#before-submitting-an-enhancementfeature-request)
+  * [Steps to ensure a good enhancement/feature suggestion](#steps-to-ensure-a-good-enhancementfeature-suggestion)
+
+[Opening a pull request](#opening-a-pull-request)
+  * [Must read](#must-read)
+  * [Good to know](#good-to-know)
+
+
+# Submitting an issue/bug report
+
+This section guides you through submitting an issue/bug report for qBittorrent.
+
+Following these guidelines helps maintainers and the community understand your report, reproduce the behavior, and find related reports.
+
+Make sure to follow these rules carefully when submitting a bug report. Failure to do so will result in the issue being closed.
+
+## Bug reporting etiquette
+
+*   Issues and comments on issues must always be in **English**.
+
+*   This project is supported by volunteers, do not expect "customer support"-style interaction.
+
+*   **Be patient**. The development team is small and resource limited. Developers and contributors take from their free time to analyze the problem and fix the issue. :clock3:
+
+*   Harsh words or threats won't help your situation. What's worse, your complain will (very likely) be **ignored**. :fearful:
+
+## What is and actual bug report?
```

Comment:
do you mean "What is an actual bug report?" ?

Reply:
Ok